src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/bolts/HaFlowCreateHubBolt.java

```diff
@@ -175,8 +177,11 @@ public void sendNotifyFlowStats(RemoveFlowPathInfo flowPathInfo) {
     }
 
     @Override
-    public void sendNotifyFlowMonitor(@NonNull CommandData flowCommand) {
-        //TODO implement https://github.com/telstra/open-kilda/issues/5172
+    public void sendNotifyFlowMonitor(@NonNull CommandData haFlowCommand) {
+        String correlationId = getCommandContext().getCorrelationId();
+        Message message = new CommandMessage(haFlowCommand, System.currentTimeMillis(), correlationId);
+        emitWithContext(HUB_TO_FLOW_MONITORING_TOPOLOGY_SENDER.name(), getCurrentTuple(),
```

Comment:
please do the same for HaFlowRerouteHubBolt and HaFlowUpdateHubBolt

Reply:
It is already there if I got you correctly.

Reply:
But I do not see changed HaFlowUpdateHubBolt file in the PR, And I see this todo https://github.com/telstra/open-kilda/blob/2a2164947342a5a22390dfea6abca42517204083/src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/bolts/HaFlowRerouteHubBolt.java#L182

NOTE. I have a type. not HaFlowUpdateHubBolt but HaFlowPathSwapHubBolt